deploy/nl/embeddings.yaml

```diff
@@ -1,30 +1,95 @@
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_sdg_2023_12_26_10_03_03.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  undata_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_undata_2024_03_20_11_01_12.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  bio_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_bio_2024_03_19_16_39_03.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  medium_lance_ft:
+    store: LANCEDB
+    embeddings: gs://datcom-nl-models-dev/lancedb/lancedb_embeddings_medium_2024_04_09_07_22_21.ft_final_v20230717230459.all-MiniLM-L6-v2
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  medium_vertex_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_medium_2024_03_14_16_38_53.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: dc-all-minilm-l6-v2-model
+  dc-all-minilm-l6-v2:
+    store: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    index_endpoint_root: 302175072.us-central1-496370955550.vdb.vertexai.goog
+    index_endpoint: projects/496370955550/locations/us-central1/indexEndpoints/8500794985312944128
+    index_id: dc_all_minilm_l6_v2_ft_1709655496660
+    model: dc-all-minilm-l6-v2-model
+  uae-large-v1:
+    store: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    index_endpoint_root: 302175072.us-central1-496370955550.vdb.vertexai.goog
+    index_endpoint: projects/496370955550/locations/us-central1/indexEndpoints/8500794985312944128
+    index_id: uae_large_v1_1709655438364
+    model: uae-large-v1-model
+  sfr-embedding-mistral:
+    store: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    index_endpoint_root: 302175072.us-central1-496370955550.vdb.vertexai.goog
+    index_endpoint: projects/496370955550/locations/us-central1/indexEndpoints/8500794985312944128
+    index_id: sfr_embedding_mistral_1709682235285
+    model: sfr-embedding-mistral-model
+
+models:
+  dc-all-minilm-l6-v2-model:
+    type: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    prediction_endpoint_id: "8518340991868993536"
+  uae-large-v1-model:
+    type: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    prediction_endpoint_id: "4910007712298827776"
+  sfr-embedding-mistral-model:
+    type: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    prediction_endpoint_id: "224012300019826688"
+  cross-encoder-ms-marco-miniilm-l6-v2:
+    type: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    prediction_endpoint_id: "3977846152316846080"
+  cross-encoder-mxbai-rerank-base-v1:
+    type: VERTEXAI
+    project_id: datcom-website-dev
+    location: us-central1
+    prediction_endpoint_id: "284894457873039360"
```

Comment:
Where do we say ranking vs. embedding?

Reply:
ah, we do not. I wonder if we should remove these ranking ones from embeddings.yaml since they're not really for embeddings & maybe non-embeddings models can have their own yaml file or something?

or we can have another field like `usage` with values EMBEDDINGS or RERANKING

wdyt

Reply:
Having another yaml seems a bit unnecessary (I think of this as all the models and indexes for NL).

Having a `usage` with that value SG.  I also have draft PR to move reranking to nl_server, which can use this loaded model.

Reply:
done